### ISSUE TITLE
"Screenshot" option for (higher resolution) graph canvas export

### ISSUE THREAD

**Issue by myxie** [ICRAR/EAGLE#686]
If I want to get an image of the a logical graph I've created, I have to take a screenshot of the browser. If I have a smaller/lower-resolution monitor, I'm not going to get a nice demonstration of the EAGLE graph, which is a shame given how slick it looks in the online editor. 

As we look toward actively promoting EAGLE/DALiuGE to users, and putting together a publications, it would be great to be able to 'export' an image of the logical graph (including or excluding the palette toolbars) at a nice resolution. 

I have no idea how difficult this will be, so feel free to say this is a pipe dream!

**Comment by james-strauss-uwa** [comment]
Thats a good idea.

In fact we used to have this functionality back when we used the GoJS library to handle the graph rendering. Then when we moved to using SVG for the graph, we ran into a problem where rendering the SVG to a PNG would not render the icons, so eventually we removed the screenshot button. But it should be possible now that we've moved to a html-based renderer.

We could investigate adding it back. I've created an issue in JIRA: [EAGLE-1227](https://icrar.atlassian.net/browse/EAGLE-1227)
